libs/service-portal/documents/src/components/DocumentLine/DocumentLine.tsx

```diff
@@ -131,22 +127,27 @@ export const DocumentLine: FC<Props> = ({
           })
         } else {
           displayPdf(docContent)
+          if (onError) {
+            onError(undefined)
```

Comment:
?

Reply:
onError is only available when documentLine is interactive. So on complete it will clear the error.